experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -756,16 +903,17 @@ export class HttpInstrumentation extends InstrumentationBase<HttpInstrumentation
     request: http.IncomingMessage,
     response: http.ServerResponse,
     span: Span,
-    metricAttributes: Attributes,
+    oldMetriceAttributes: Attributes,
+    stableMetriceAttributes: Attributes,
```

Comment:
small typo here (metrice/metrics) :slightly_smiling_face:

Reply:
metrice is metrics french cousin 😜

I'll fix it